keras/constraints.py

```diff
@@ -11,29 +11,62 @@ def get_config(self):
 
 
 class MaxNorm(Constraint):
-    def __init__(self, m=2):
+    '''Constrain the weights incident to each hidden unit to have a norm less than or equal to a desired value.
```

Comment:
I've never heard "hidden unit" refer to a single weight. I was trying to match the language from the reference paper, i.e. "In other words, if w represents the vector of weights incident on any hidden unit, the neural network was optimized under the constraint ||w||2 ≤ c. " What would be more clear?